### DISCUSSION TITLE
Change 'action bar' on Data view to the top of the window?

### DISCUSSION THREAD

**Discussion by pjfsilva** [dbeaver/dbeaver#21222] [Q&A] [answered]
Is it possible to change the action bar (create new row, save button, etc) on the Data view of a table, to the top, after or before the filter bar somehow?

![image](https://github.com/dbeaver/dbeaver/assets/2199803/16b55042-32dd-4dd3-9aaf-67d1d7c63b68)

Basically take most or all the actions marked by the red arrow to the blue part of the screen?

**Comment by E1izabeth** [comment, marked as answer]
It's not possible now, but you could create a feature request.
